README.md

```diff
@@ -1,24 +1,26 @@
-# Open Urban Mapping project
+# The Open Urban Mapping project
 
 Introduction
 -------------
-Now it's possible to digitize all buildings in the World using Computer vision and Satellite imagery. This project is started to help accelerate the routine work of digitizing of the buildings and to fill the data gaps in Openstreetmap's open data.  
+Whenever we use a digital map to look for the closest supermarket or find the fastest way home, buildings are one of the key elements that pop up on our screen. More often than not, the object of our search is located _in_ a certain building, be it our apartment, our favorite store or the office that we work in. Imagine there is no visual representation of that building on the map. Our office, shop or apartment may have coordinates of their own, but, arguably, we don't think in terms of coordinates. Our eyes can't _(yet?)_ calculate the latitude and longitude of what we're looking at, but they do a pretty good job recognizing the building from the map in reality.  
```

Comment:
Better to use in a webpage. At start here it should be the sort statement regarding project goals.